stable/stolon/templates/create-cluster-job.yaml

```diff
@@ -0,0 +1,50 @@
+{{ if .Release.IsInstall }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ template "stolon.fullname" . }}-create-cluster
+  labels:
+    app: {{ template "stolon.name" . }}
+    chart: {{ template "stolon.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+  annotations:
+    "helm.sh/hook-delete-policy": hook-succeeded
+spec:
+  template:
+    metadata:
+      labels:
+        app: {{ template "stolon.fullname" . }}
+        release: {{ .Release.Name }}
+      annotations:
+        {{- if eq .Values.store.backend "etcd" }}
+        pod.beta.kubernetes.io/init-containers: '[
+              {
+                  "name": "create-cluster",
+                  "image": "gcr.io/google_containers/etcd-amd64:3.0.14",
+                  "command": ["/bin/sh", "-c", "while ! etcdctl --endpoints {{ .Values.store.endpoints }} cluster-health; do sleep 1 && echo -n .; done"],
+                  "imagePullPolicy": {{ .Values.image.pullPolicy | quote }}
+              }
+          ]'
+        {{- end }}
+    spec:
+      restartPolicy: OnFailure
+      serviceAccountName: {{ template "stolon.serviceAccountName" . }}
+      containers:
+      - name: {{ template "stolon.fullname" . }}
```

Comment:
Please make list indentation consistent throughout the chart. Since you mostly indent two spaces, please do so as well here.

Reply:
fixed